lambda/localizationLambda.js

```diff
@@ -10,7 +10,7 @@
  * If you are viewing this file inside of AWS, this code comes from https://github.com/sharesight/www.sharesight.com/.
  */
 
-const version = '5';
+const version = '8';
```

Comment:
We were on 5 even?  Weird..

Reply:
I don't think there is an easy way to get this value from the context or update it automatically.